frontend/cypress/integration/queryBuilder.spec.js

```diff
@@ -168,7 +168,19 @@ describe('The Query Builder', () => {
     // TODO: create test
   });
 
-  xit('makes a copy of a dataset', () => {
-    // TODO: create test
+  it('makes a copy of a dataset', () => {
+    cy.fixture('copyDataset').then((dataset) => {
+      cy.intercept('api/datasets/mine/', dataset);
+    });
+    cy.visit('/');
+    cy.get('[data-testid="dataset-action"]').first().click({ force: true });
```

Comment:
This won't do as you'll always need to create new dataset fixture files just to make sure there's only one returned ... intercept and directly navigate to the actual dataset edit page. Refer to the test on line 83 (sure u can do it better)

Reply:
I had done that at first but I was getting issues,let me see how to work around it